packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header-item.scss

```diff
@@ -10,24 +11,30 @@
 }
 
 .abnormal-value {
-  border: 1px solid $ui-05;
-  background-color: white;
+  border: solid 1px #ffc69e;
+  background-color: #fff2e8;
   display: flex;
   flex-direction: column;
   justify-content: center;
   width: fit-content;
-  padding: 0rem 0.25rem;
+  gap: layout.$spacing-02;
+  padding: layout.$spacing-02;
 }
 
 .critical-value {
   @extend .abnormal-value;
-  border: 2px solid $danger;
+  border: solid 1px #da1e28;
```

Comment:
Thanks @Twiineenock , do these color exist in the styleguide ?

Reply:
Thanks @CynthiaKamau, yes they do exist. Therefore I have commited with this change!